OCR/frontend/src/componets/ExtractUploadFile.tsx

```diff
@@ -10,15 +10,68 @@ interface IFilesObj {
   files: File[];
 }
 
+interface Template {
+  name: string;
+  description: string;
+  pages: {
+    image: string;
+    fieldNames: string[];
+  }[];
+}
+
 export const ExtractUploadFile: React.FC<ExtractUploadFileProps> = ({
   onUploadComplete,
 }) => {
   const id = useId();
   const { addFile } = useFiles();
   const [template, setTemplate] = useState<string>("");
   const [uploadProgress, setUploadProgress] = useState<number>(0);
+  const [templates, setTemplates] = useState<Template[]>([]);
   const [uploadedFile, setUploadedFile] = useState<File | null>(null);
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const loadTemplatesFromLocalStorage = () => {
+    const storedTemplates = localStorage.getItem("templates");
+    if (storedTemplates) {
+      const parsedTemplates = JSON.parse(storedTemplates);
+      setTemplates(parsedTemplates);
+    } else {
+      setTemplates([]);
```

Comment:
maybe only load the fake ones here when there aren't valid ones.

Reply:
Awesome that works. I've made that change.